stripe/src/main/java/com/stripe/android/model/SourceOwner.java

```diff
@@ -59,64 +59,64 @@
         mVerifiedPhone = verifiedPhone;
     }
 
-    SourceAddress getAddress() {
+    public SourceAddress getAddress() {
         return mAddress;
     }
 
-    void setAddress(SourceAddress address) {
-        mAddress = address;
+    public String getEmail() {
```

Comment:
Will need to be able to call getters on this object, so I made the methods public. After making them public, I moved them above the package-private methods as a matter of style.